crsf.md

```diff
@@ -1121,6 +1122,28 @@ MSP frame over CRSF Payload packing:
 
 ## **0x80 Ardupilot reserved**
 
+## **0x81, 0x82 mLRS reserved**
+
+The 0x80 and 0x81 frames establish the communication of mLRS TX modules with mLRS Configuration Lua scripts running on e.g. EdgeTx and OpenTx remote controllers. They encapsulate mLRS 'mBridge' protocol messages in CRSF frames. The protocol enables the setting of parameters and other functionality, but especially also communicates meta data which are needed for mLRS' parameter model, providing various information to the user, version control, and more features specific to the mLRS link system.
+
+**0x80**: Communication from Lua script/remote controller to TX module. The frame format is:
+
+```mermaid
+flowchart LR
+    0xEE ~~~ id0[Frame Length] ~~~ 0x80 ~~~ 0x4F ~~~ 0x57 ~~~ id1[Command Byte] ~~~ Payload ~~~ CRC
+```
+
+**0x81**: Communication from TX module to Lua script/remote controller. The frame format is:
+
+```mermaid
+flowchart LR
+    0xEA ~~~ id0[Frame Length] ~~~ 0x81 ~~~ id1[Command Byte] ~~~ Payload ~~~ CRC
+```
+
+The Frame Length and CRC are as per the CRSF specification in the above. The mBridge protocol is relatively complicated, as it serves mLRS for various additional purposes besides communication with the Lua configuration script. 
```

Comment:
I assume this means that you won't document these here. Can you provide a more precise link to where they are documented in your project?

Reply:
I frankly think that, for open source projects, a level of "documentation by source" needs to be accepted. If you ask ArduPilot to provide a complert full description this will blow up your doc. So it is here.

Reply:
I read theirs as telemetry messages, where as here I see them as commands. But we can get this in for now and if there's anyone in the community that wishes to explain it better, we're open for PRs.